tests/unit/test_simulations.py

```diff
@@ -111,7 +111,7 @@ def test_simulations_output_file(tmpdir, passive, empty_file, sim):
         "delta": 0.04,
         "p_swap": 0.5,
         "trials": 100,
-        "passive": True,
+        "passive": passive,
```

Comment:
This doesn't make sense to me at all, maybe I am missing something. Just to get things going, I'll revert it back with a comment and we can discuss it soon.